src/Raku/ast/expressions.rakumod

```diff
@@ -1211,7 +1211,7 @@ class RakuAST::MetaInfix
   is RakuAST::CheckTime
 {
     method IMPL-HOP-INFIX() {
-        self.get-implicit-lookups().AT-POS(0).resolution.compile-time-value()(
+        self.get-implicit-lookups()[0].resolution.compile-time-value()(
```

Comment:
Both are kinda wrong, though [0] is even more so. get-implicit-lookups is a public API and as such can be overridden by a Raku class which would return a List, not a VMArray. Thus we really should IMPL-UNWRAP-LIST the result before indexing into it.